pkg/config/config.go

```diff
@@ -213,6 +213,22 @@ func (c *Config) Validate() error {
 	return nil
 }
 
+// Generate config clone.
```

Comment:
Just to follow Go conventions.
The comment should be formatted by the function name `Clone ...`

```suggestion
// Clone generates a cloned config object.
```